### ISSUE TITLE
Remove or relocate Mastodon verification link

### ISSUE THREAD

**Issue by chalin** [open-telemetry/opentelemetry.io#2104]
_Originally posted by @chalin in https://github.com/open-telemetry/opentelemetry.io/issues/2028#issuecomment-1320211835_

> Once the verification is done, will this be removed? If not, I'd suggest that we put it somewhere other that that base layout file.

**Comment by chalin** [comment]
If it needs to be kept, does it need to appear in every site file or only the homepage? /cc @austinlparker

**Comment by austinlparker** [comment]
I don't know if it re-checks periodically, but it only needs to be on the homepage.

**Comment by chalin** [comment]
Thanks for confirming @austinlparker. Do you know if it needs to be visible or only present in the homepage HTML?

**Comment by austinlparker** [comment]
It needs to be present... I didn't think it was visible right now? It's just an empty link.

**Comment by chalin** [comment]
Right, I just scratched out my question when I noticed it was empty 🙃. Thx!

**Comment by svrnm** [comment]
- See #2115

**Comment by chalin** [comment]
For more context see, https://github.com/open-telemetry/opentelemetry.io/pull/2115#issuecomment-1357717511.
